plugins/playlist/src/components/PlaylistPage/PlaylistHeader.tsx

```diff
@@ -83,6 +83,7 @@ export const PlaylistHeader = ({ playlist, onUpdate }: PlaylistHeaderProps) => {
   const rootRoute = useRouteRef(rootRouteRef);
   const [openEditDialog, setOpenEditDialog] = useState(false);
   const [openDeleteDialog, setOpenDeleteDialog] = useState(false);
+  const [popupMessage, setPopupMessage] = useState('');
```

Comment:
Don't put this in a state. Just call `alertApi.post` whenever you want, directly in code. Otherwise you'll unnecessarily re-render this component twice (once when setting the message and once after clearing it again)